tests/testthat/test-add_resource.R

```diff
@@ -85,6 +85,7 @@ test_that("add_resource() returns error on invalid or empty data frame", {
 })
 
 test_that("add_resource() returns error if CSV file cannot be found", {
+  testthat::skip_if_offline()
```

Comment:
@damianooldoni does this apply to the whole test or just the next assertion? Because currently (line 117) it is listed twice.

Reply:
You are right! it's a duplicate. If you remove your internet connection and run the test, you skip the very same number of tests (6). Fixed in https://github.com/frictionlessdata/frictionless-r/commit/74de9c90aff02a4be208d2ed696d76aa3df0bb06

Reply:
I also removed a number of other skip_if_online() tests that weren't needed.